scalableminds/webknossos
# Reactify Task List

I reactified the task list and got rid of the weird query interface and replaced it with a regular form.

### Mailable description of changes:
- [Admin.Task List ] Modernized table and added more filtering / sorting options
- [Admin.Task List ] Replaced complicated "task query" search interface with form
- [Admin.Task List ] Enabled searching for active tasks for specific users

### Steps to test:
- Go to Admin --> tasks
- Search for tasks and double check if the response is correct (e.g. double check with your DB)

### Issues:
- fixes #1384 

------
